token-swap/js/src/index.ts

```diff
@@ -428,11 +428,16 @@ export class TokenSwap {
       curveParameters,
     );
 
+    const options = {
+      skipPreflight: false,
+    };
```

Comment:
Instead of hard-coding these everywhere, we can add `options: ConfirmOptions` as a parameter to all of these functions, similar to how the token JS bindings do it.  Here's an example for `createMint`: https://github.com/solana-labs/solana-program-library/blob/dd8f4d138fe5c51067737124cf8feb47daf75254/token/js/src/actions/createMint.ts#L36

This might cause more churn since you'll also have to update the tests calling these functions, but it'll help functionality a lot

Reply:
Made the changes!